distribution/changelog.txt

```diff
@@ -17,6 +17,7 @@
 - Fix: [#18169] CJK, Arabic and Vietnamese display all text as ‘???’ on Android.
 - Fix: [#18309] Flying and Multi Dimension trains glitch when changing between inverted and uninverted track when uncap fps is on.
 - Fix: [#19506] Queue paths can be placed on level crossings by replacing an existing regular path.
+- Fix: [#21908] Ride mode warnings when hovering track designs.
```

Comment:
This issue is also reported fixed in the changelog for v0.4.14. The fix in question turned out to be a partial fix. Are we fine with both lines existing? Just checking if there's a policy.

Reply:
I don’t think we have a standard procedure for this. It should at least be in the changelog for this version, I’d say - not sure what to do with the older line.